server.js

```diff
@@ -2600,15 +2600,24 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // GitHub release-download-count integration.
-camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)(\/[^\/]+)?\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
   var user = match[1];  // eg, qubyte/rubidium
   var repo = match[2];
-  var tag = match[3];
+
+  var tag = match[3]; //null for all releases
   var asset_name = match[4].toLowerCase(); // eg. total, atom-amd64.deb, atom.x86_64.rpm
   var format = match[5];
-  var release_path = tag !== 'latest' ? 'tags/' + match[3] : 'latest';
-  var apiUrl = 'https://api.github.com/repos/' + user + '/' + repo + '/releases/' + release_path;
+  var total = true;
+  if (tag) {
+    total = false;
+  }
+
+  var apiUrl = 'https://api.github.com/repos/' + user + '/' + repo + '/releases';
+  if (!total) {
+    var release_path = tag !== 'latest' ? 'tags/' + tag : 'latest';
```

Comment:
I logged the values of tag and release_path, and tag was `"/latest"`, and so release_path was `"tags//latest"` (which still works, as most servers convert double slashes into single slashes), but not `"latest"`, as the code presumably intends to do.

Either we must `tag = tag.slice(1)` somewhere, or we must compare `tag` to things that have a leading slash (ie, `tag !== '/latest'`, and `'tags' + tag`).